cmd/crank/beta/convert/pipelinecomposition/converter.go

```diff
@@ -72,21 +72,6 @@ func convertPnTToPipeline(c *v1.Composition, functionRefName string) (*v1.Compos
 		Resources: []v1.ComposedTemplate{},
 	}
 
-	// Most EnvironmentConfig settings remain at the Composition Level, but
-	// Environment Patches are handled at the Function level
-	if c.Spec.Environment != nil {
-		cp.Spec.Environment = &v1.EnvironmentConfiguration{
-			DefaultData:        c.Spec.Environment.DefaultData,
-			EnvironmentConfigs: c.Spec.Environment.EnvironmentConfigs,
-			Policy:             c.Spec.Environment.Policy,
-		}
-		if len(c.Spec.Environment.Patches) > 0 {
-			input.Environment = &v1.EnvironmentConfiguration{
-				Patches: c.Spec.Environment.Patches,
-			}
-		}
-	}
-
```

Comment:
@stevendborrelli I'm not sure what to do about this converter code.

As we remove the native support easy migration will be more important than ever. For now I've deleted the migration code that pertains to environment configs just because it's so tightly coupled to the environment config types that we want to delete.

Perhaps we could move all those types here into the migration command's code tree? I worry though if we do that they'll sit around forever. Another option (that I'm currently leaning toward) is that we tell folks they need to use `crank` v1.17 or earlier to do the migration.

Reply:
as discussed we can drop this and either add a new command or update this one in a follow-up pr to migrate to `function-environment-configs`.